bin/tessel-2.js

```diff
@@ -19,9 +19,13 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
 
 // Check for updates
 const pkg = require('../package.json');
-updateNotifier({
-  pkg
-}).notify();
+
+try {
+  updateNotifier({ pkg }).notify();
+} catch (err) {
+  logs.warn('Failed to check for newer version of the CLI');
```

Comment:
Sounds good!

What we could to would be to only show the error the first time, if only there was some kind of library that made it very easy to store if we have previously shown the error or not... Oh, I know! Nah, just kidding ;)